pkg/bookings/bookings.go

```diff
@@ -0,0 +1,21 @@
+package bookings
+
+import (
+	"context"
+
+	"github.com/clearchanneloutdoor/io-sdk-golang/pkg/clients"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+const (
+	serverUrl string = "https://direct.cco.io"
+)
+
+func NewClient(ctx context.Context, oauth2 *clientcredentials.Config, overrideSvr ...string) (*clients.Client[Booking], error) {
```

Comment:
don't forget to reference these new clients in cmd/main.go

Reply:
oops i'll add that in now!